lumberjack_test.go

```diff
@@ -202,11 +208,13 @@ func TestMaxBackups(t *testing.T) {
 	dir := makeTempDir("TestMaxBackups", t)
 	defer os.RemoveAll(dir)
 
+	notify := make(chan struct{})
 	filename := logFile(dir)
 	l := &Logger{
-		Filename:   filename,
-		MaxSize:    10,
-		MaxBackups: 1,
+		Filename:      filename,
+		MaxSize:       10,
+		MaxBackups:    1,
+		notifyRemoved: notify,
```

Comment:
Same as earlier - consider initialising inline and calling `waitForNotify(l.notifyRemoved, t)` below.